.travis.yml

```diff
@@ -4,10 +4,13 @@ services: docker
 env:
   - VERSION=11.0 VARIANT=fpm
   - VERSION=11.0 VARIANT=apache
+  - VERSION=11.0 VARIANT=alpine-fpm
```

Comment:
Also you should use `update.sh` to generate the Dockerfiles and update `.travis.yml`